Cargo.toml

```diff
@@ -35,6 +35,7 @@ global-context = ["std"]
 # if you are doing a no-std build, then this feature does nothing
 # and is not necessary.)
 global-context-less-secure = ["global-context"]
+core-error = []
```

Comment:
If we're going to use this, I'd want the feature to be named `unstable-nightly-core-error` and document the limitations.